docs/pages/database-access/guides/azure-postgres-mysql.mdx

```diff
@@ -8,17 +8,18 @@ description: How to configure Teleport Database Access with Azure Database for P
   opened={true}
   scope={["oss", "enterprise"]}
   scopeOnly={true}
-  min="8.1"
 >
-  Static configuration of database access for Azure PostgreSQL/MySQL is available starting from Teleport
-  `8.1` and Azure database auto-discovery is available starting from Teleport `10.2`.
-</Details>
+Static configuration of database access for Azure PostgreSQL/MySQL is available
+starting from Teleport `8.1` and Azure database auto-discovery is available
```

Comment:
Given that the docs for versions older than v8 are not even available on our docs site anymore, I think it's probably safe to remove the note about 8.1.

Reply:
Removed.